src/main/java/hudson/plugins/git/extensions/impl/MessageExclusion.java

```diff
@@ -23,6 +23,7 @@
  * @author Kanstantsin Shautsou
  */
 public class MessageExclusion extends GitSCMExtension {
+
```

Comment:
He'd be right to in this case: no reason to have a diff in this file at all, just a whitespace oversight when backing out something.